samples/dashboard/demos/climate/demo.js

```diff
@@ -272,19 +269,39 @@ async function setupDashboard() {
             }],
             title: 'KPI 1'
         }, {
-            cell: 'kpi-chart',
+            cell: 'city-chart',
             type: 'Highcharts',
             chartOptions: {
                 chart: {
-                    type: 'line',
-                    zooming: {
-                        type: 'x'
-                    }
+                    events: {
+                        render: function () {
+
+                            if (!this.styledMode) {
+                                return;
+                            }
+
+                            // force point colors
+                            for (const point of this.series[0].points) {
+                                if (point.graphic && point.color) {
+                                    point.graphic.element.style.fill =
+                                        point.color;
+                                }
+                            }
+                        }
+                    },
```

Comment:
Why is this hack necessary? It's used twice in this demo. Styled mode should work with `colorAxis` out of the box, like seen for example in http://localhost:3030/samples/#view/highcharts/css/grid-light-highmaps.

If it doesn't, it's a bug that should be reported and fixed. Maybe this is an issue with mappoint and scatter series types?

Reply:
#14114

Reply:
PR: https://github.com/highcharts/highcharts/commit/ea30b04d4f873cf575ca0587af217deec1c96eac